autopilot/Wine-Reviews/Autopilot Does Text.ipynb

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
<p>It looks like customers are required to have a Kaggle account to download this dataset. Is this something we want? Is this an open source dataset that we can download from elsewhere? 
</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1421/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
It's a non-commercial license too.

Reply:
I think it might be ok but you'll need to have the license and attribution in this folder / readme.